app/Food/food.py

```diff
@@ -0,0 +1,8 @@
+class Food(object):
+    def __init__(self, name, expiration_date, use_by_length):
+        self.name = name
+        self.expiration_date = expiration_date
+        self.use_by_length = use_by_length
+    
+    def open_date():
```

Comment:
I think this should just be called "open" or "openItem"

Reply:
Done.